src/components/Chip.js

```diff
@@ -151,20 +156,20 @@ class Chip extends React.Component<Props, State> {
 
     const borderColor =
       mode === 'outlined'
-        ? color(dark ? white : black)
+        ? color(selectedColor || color(dark ? white : black))
```

Comment:
seems there is double wrapping of color. change to:

```suggestion
        ? color(selectedColor !== undefined ? selectedColor : dark ? white : black)
```